### PR TITLE
Apexnote

### PR DESCRIPTION
**Pull request by rminnich**
This fixes the apex regression test. You can try on apex if you want.

### CODE REVIEW

File: sys/src/regress/apexnotecont.c [hunk 1]
```diff
@@ -28,12 +28,19 @@ _notetramp(int sig, void (*hdlr)(int, char*, struct Ureg*), struct Ureg *u)
...
 	p->restorepc = u->ip;
+print("b\n");
 	p->sig = sig;
+print("c\n");
 	p->hdlr = hdlr;
+print("d\n");
 	p->u = u;
+print("e\n");
 	nstack++;
+print("f\n");
```
Comment (dancrossnyc) [inline sys/src/regress/apexnotecont.c]: Are all of these print()'s still necessary? This feels like debugging code that leaked into a commit.

Comment (rminnich) [inline sys/src/regress/apexnotecont.c]: It's a regression test so I think we should leave them in.

Comment (dancrossnyc) [inline sys/src/regress/apexnotecont.c]: Okay; that makes sense.

Comment (rminnich) [inline sys/src/regress/apexnotecont.c]: I don't know I might be full of it ... is it ok in a test situation to allow arbitrary text on stdout? It's super handy in this case for isolating where the failure is but ... your ideas here are most welcome. Put it on stdout, stderr, or ...

**Pull request merged**